docs/modules/ROOT/pages/how-tos/appcat/vshn/postgres/manual-restore.adoc

```diff
@@ -0,0 +1,168 @@
+Given the bucket from previous step, access the S3 Bucket with s3cmd or any similar S3 tool. Make sure the data is present.
+
+* Configure
++
+[source,bash]
+----
+s3cmd --configure
+----
+
+* Show Data
++
+[source,bash]
+----
+s3cmd ls --recursive s3://<bucket-name>
+----
+
+=== Step 2.3 - Create SGBackup custom resource
+In case there are no `SGBackup` resources in the instance namespace then one should be created for a proper restore.
+The following is a template for the said resource:
+
+[source,yaml]
+----
+apiVersion: stackgres.io/v1
+kind: SGBackup
+metadata:
+  name: manual-backup
+  namespace: <instance-namespace> # <1>
+spec:
+  sgCluster: <sgcluster-resource-name> # <2>
+status:
+  backupInformation:
+    postgresVersion: "150005" # <3>
+  backupPath: sgbackups.stackgres.io/example-name/example-name/15 # <4>
+  internalName: base_00000048000000010000009D # <5>
+  process:
+    status: Completed
+  sgBackupConfig:
+    baseBackups: {}
+    storage:
+      s3Compatible:
+        awsCredentials:
+          secretKeySelectors:
+            accessKeyId:
+              key: AWS_ACCESS_KEY_ID
+              name: <bucket-secret-credentials> # <6>
+            secretAccessKey:
+              key: AWS_SECRET_ACCESS_KEY
+              name: <bucket-secret-credentials> # <6>
+        bucket: <bucket-name> # <7>
+        enablePathStyleAddressing: true
+        endpoint: https://objects.lpg.cloudscale.ch # <8>
+        region: lpg <9>
+      type: s3Compatible
+----
+<1> The instance namespace from where the database is running
+<2> `SGCluster` resource from the instance namespace. It has the same name as the XVSHNPostgreSQL resource
+<3> The database version can be retrieved as follows:
++
+[source,bash]
+----
+kubectl -n <instance-namespace> get SGCluster <name>
+----
++
+The format is `major version + 00 + minor version`. Example: `15 (major) + 00 + 05 (minor) = 150005`
+<4> The path in the S3 bucket, from the root folder to the version of the database instance:
++
+[source,bash]
+----
+s3cmd ls --recursive s3://odoo-database-9msf4 | awk '{print $4}' | grep basebackups | tail -1 | cut -d "/" -f 4,5,6,7
+----
+<5> Last base backup code from the S3 bucket in the format `base_hexcode`:
++
+[source,bash]
+----
+s3cmd ls --recursive s3://odoo-database-9msf4 | awk '{print $4}' | grep basebackups | tail -1 | cut -d "/" -f 9 | cut -d "_" -f 1,2
+
+----
+<6> Secret of the S3 bucket. Can be retrieved from the XObjectBucket which has the same name as XVSHNPostgreSQL.
+In case there is no secret it has to be recreated with the credentials from the cloud provider.
++
+[source,bash]
+----
+kubectl get XObjectBucket --as cluster-admin <name>
+----
+<7> THe bucket name
+<8> S3 Cloud provider endpoint
+<9> Region of the bucket
+
+== Step 3 Restore the Database
+The restore process is straightforward. The process is to tell to the PostgreSQL cluster to use our backup.
+Update `SGCluster` resource with the following snippet of code:
```

Comment:
We'd have to re-create the object though, the `initialData` field only triggers on the very first startup of the cluster AFAIK.

Reply:
True, we have to recreate the object. I will amend the changes.